app/uk/gov/hmrc/icedsubscriptionfrontend/views/LandingPage.scala.html

```diff
@@ -0,0 +1,78 @@
+@*
```

Comment:
can we use the components provided by [play-frontend-hmrc](https://github.com/hmrc/play-frontend-hmrc) to create the page using accessible components, there is this [chrome extension](https://github.com/hmrc/play-frontend-govuk-extension) that can help with the twirl